app/lib/userPreferences.ts

```diff
@@ -7,11 +7,12 @@ const MMKV = new MMKVStorage.Loader()
 	.initialize();
 
 class UserPreferences {
+	mmkv: MMKVStorage.API;
```

Comment:
WDYT if we create this field as private ?
`private mmkv: MMKVStorage.API;`

Reply:
okays :)